### PR TITLE
Sort context names alphabetically

### PR DESCRIPTION
**Pull request by rubenaprikyan**
- Show contexts list ordered by stringified value alphabetically

### CODE REVIEW

**Review by Hamik25 — APPROVED** [review]
Looks good!

**Review by arsengit — APPROVED** [review]

File: aim/web/ui/src/services/models/runs/util.ts [hunk 1]
```diff
@@ -47,6 +48,16 @@ export function getMenuItemFromRawInfo(
 
   const data: IMenuItem[] = [];
 
+  const sortOrder = alphabeticalSortComparator({
+    orderBy: 'name',
+    additionalCompare: (name1: string, name2: string) => {
+      if (name2 === 'EMPTY CONTEXT') {
+        return 0;
```
Comment (roubkar) [inline aim/web/ui/src/services/models/runs/util.ts:55]: 0 or -1 ?

**Comment by rubenaprikyan** [comment]
Resolves #1122

File: aim/web/ui/src/services/models/runs/util.ts (hunk 1 above)
Comment (rubenaprikyan) [reply aim/web/ui/src/services/models/runs/util.ts:55]: It need to show at the beginning of the list.
In the menu list of TraceVisualizer if there is an empty context, it is always positioned at the beginning of the list, so it need to keep position the same.

Please also see the comment in line 108, 109 of this file

**Review by roubkar — APPROVED** [review]

**Pull request merged**